src/parser/symtable.ts

```diff
@@ -1299,9 +1295,9 @@ export class SymbolTable {
            operation.  It is different than a->name when a->name is a
            dotted package name (e.g. spam.eggs)
         */
-        const name = a.asname === null ? a.name : a.asname;
+        const name = a.asname ?? a.name;
         const dot = name.indexOf(".");
-        let storeName = null;
+        let storeName: string;
```

Comment:
I didn't know this was allowed, shouldn't this be `string|null` now?

Reply:
No need. Because typescript infers that the variable will be set to a string and isn't used otherwise.